react/tic-tac-toe/src/app/screens/Game/styles.module.scss

```diff
@@ -1,6 +1,7 @@
 .game {
   display: flex;
   flex-direction: row;
```

Comment:
Just a quick note. There is no need to change this, but in wolox we have generic classes for these type of things like `row` and `column`